cli/cmd/transformCmd.go

```diff
@@ -39,6 +40,8 @@ func init() {
 
 	transformCmd.Flags().StringVarP(
 		&input, "input", "i", "", "input file (optional; if not specified, stdin/pipe is used)")
+	transformCmd.Flags().BoolVarP(
+		&ndjson, "ndjson", "", false, "change the output format to ndjson")
```

Comment:
@jf-tech is this what you meant with a command line option or did you want something like `--format`?